index.html

```diff
@@ -16,13 +16,10 @@
             <a>Prestations</a>
             <a>Contact</a>
           </div>
-          
           <footer>
             <div class="detail">
               <div class="mention">
                 <a href="mentionsLegales.html" target="_blank">Mentions légales</a>
-                <a href="CGU" target="_blank">CGU</a>
-                <a href="PolitiqueDeConfidentialite" target="_blank">Politique de confidentialité</a>
```

Comment:
Why did you remove this ?

Reply:
un sage nous a dit, il y a peu , qu'on n'avait pas besoin de cgu et de politique de confidentialité pour une page non commerciale

Reply:
a wise man told us recently that we did not need a cgu or privacy policy for a non-commercial page